Formula/s/sysdig.rb

```diff
@@ -73,6 +74,10 @@ class Sysdig < Formula
   def install
     (buildpath/"falcosecurity-libs").install resource("falcosecurity-libs")
 
+    # fix `libzstd.so.1: error adding symbols: DSO missing from command line` error
+    # https://stackoverflow.com/a/55086637
+    ENV.append "LDFLAGS", "-Wl,--copy-dt-needed-entries" if OS.linux?
+
```

Comment:
Can move this along to make its dependencies ✅. But would be nice to figure out how this happened and how to properly fix it.